src/main/java/uk/gov/pay/directdebit/mandate/model/GoCardlessMandate.java

```diff
@@ -37,6 +38,10 @@ public String getGoCardlessReference() {
         return goCardlessReference;
     }
 
+    public GoCardlessCreditorId getGoCardlessCreditorId() {
```

Comment:
If this could be `null` (and I think legacy GoCardless mandates in the database mean it might be), we may wish to return an `Optional<GoCardlessCreditorId>` here.

Reply:
As discussed this cannot be null in real case, but only in our existing GC mandates. We are going to create another DB migration to initialise the existing creditor id with dummy value and then remove it in this PR